src/main/java/io/thinkit/edc/client/connector/model/VerifiableCredentialResource.java

```diff
@@ -0,0 +1,133 @@
+package io.thinkit.edc.client.connector.model;
+
+import static io.thinkit.edc.client.connector.utils.Constants.*;
+import static jakarta.json.Json.createObjectBuilder;
+
+import jakarta.json.JsonObject;
+import jakarta.json.JsonObjectBuilder;
+
+public class VerifiableCredentialResource {
```

Comment:
given that there's no json-ld hassle here, it could be modeled as a simple `record` and deserialized through a jackson `ObjectMapper`, it will save a lot of code.

Same for the other classes.